docs/terminal/melbourne.md

```diff
@@ -32,45 +33,56 @@ See also: [AV ADC Offline](#av-adc-offline).
 !!! tip
     When AV ADC is not online, consider publishing an **ATIS Zulu** for the aerodrome, to inform pilots about the airspace reclassification. The *More ATIS* plugin has a formatted Zulu ATIS message.
 
+### Avalon Approach (MAV)
+MAV is responsible for the Class D and E airspace in the AV CTR, `SFC`-`A045`.  
+They are responsible for the same portion of airspace when it is [reclassified](#av-ctr) due to AV ADC being offline.
+
+The Southern Wedge of MAV (shown below) is Class E `A015`-`A045`, and extends in to YWE(WON) airspace. YWE(WON) is responsible for the airspace **above and below** it, and it is **Class G**.
```

Comment:
The Southern Wedge of MAV (shown below) is Class E `A015`-`A045`, and extends in to YWE(WON) airspace. YWE(WON) is responsible for the **Class G** airspace **above and below** it.